libbeat/processors/add_kubernetes_metadata/kubernetes.go

```diff
@@ -251,6 +259,10 @@ func (k *kubernetesAnnotator) Run(event *beat.Event) (*beat.Event, error) {
 	if !k.kubernetesAvailable {
 		return event, nil
 	}
+	if kubernetesMetadataExist(event) {
+		k.log.Debug("Skipping add_kubernetes_metadata processor as kubernetes metadata already exist")
```

Comment:
This will be reporting the log for every single event, means that even if it is in `debug` level it could become quite overwhelming.

Reply:
Only in case user enables debug mode. In this `Run` method there are other debug messages as well. Do you think we shouldn't log this at all?

Reply:
Ok, since we have already another debug message in the main flow we can preserve this one too. Maybe in the future we can revisit the logging in this method if we see it becomes noisy, but for now we can keep it.

Reply:
Also we log each published event in `processing/processors.go:203`. So tbh the logging of one line that the processor is skipped is nothing comparing to that